src/phase0/state_transition/mod.rs

```diff
@@ -761,20 +761,21 @@ pub fn get_beacon_committee<
     slot: Slot,
     index: CommitteeIndex,
     context: &Context,
-) -> Result<&[ValidatorIndex], Error> {
+) -> Result<Vec<ValidatorIndex>, Error> {
```

Comment:
what was wrong w/ the slice here?

Reply:
we return a reference to a local variable